py/desisim/lya_spectra.py

```diff
@@ -27,6 +35,9 @@ def get_spectra(lyafile, nqso=None, wave=None, templateid=None, normfilter='sdss
           SEED.
         qso (desisim.templates.QSO, optional): object with which to generate
           individual spectra/templates.
+        add_dlas (bool): Inject damped Lya systems into the Lya forest
+          These are done according to the current best estimates for the incidence dN/dz
```

Comment:
Citation added (it was below already)

For the spline, I fear it is much more complicated than that although
I do agree with your concern.